src/features/app/index.js

```diff
@@ -0,0 +1 @@
+export * from './components';
```

Comment:
Why make a folder in features for App?
We can just place app.js with ./src/index.js.
Components folder seems unnecessary. 
WDYT?

Reply:
Because it keeps the structure consistent and other features will have multiple components. Also extensibility. If we want to add more components later, I don't want to keep changing the structure.

Reply:
Okay cool.!